packages/legend-studio/src/stores/graphModifier/DSLMapping_GraphModifierHelper.ts

```diff
@@ -74,7 +86,12 @@ export const setImpl_setRoot = action(
 
 //
 export const mapping_addClassMapping = action(
-  (mapping: Mapping, val: SetImplementation): void => {
+  (
```

Comment:
I think you missed a bunch of methods in this file, such as `enumValueMapping_setSourceValues`